webpack.config.prod.js

```diff
@@ -1,54 +1,54 @@
-const path = require('path');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 
 module.exports = {
-	mode: 'production',
-	entry: './src/index.js',
-	output: {
-		filename: '[name].bundle.js',
-		chunkFilename: '[name].bundle.js',
-		path: path.join(__dirname, '/build')
-	},
-	resolve: {
-		modules: [ path.resolve(__dirname, './src'), 'node_modules' ],
-		extensions: [ '.js', '.jsx', '.json' ]
-	},
-	devServer: {
-		allowedHosts: path.join(__dirname, './build'),
-		compress: true,
-		open: true,
-		hot: false,
-		liveReload: false
-	},
-	optimization: {
-		splitChunks: {
-			chunks: 'all'
-		}
-	},
-	module: {
-		rules: [
-			{
-				test: /\.(js|jsx)$/,
-				exclude: /node_modules/,
-				use: {
-					loader: 'babel-loader'
-				}
-			},
-			{
-				test: /\.css$/,
-				use: [ MiniCssExtractPlugin.loader, 'css-loader' ]
-			}
-		]
-	},
-	plugins: [
-		new CleanWebpackPlugin(),
-		new HtmlWebpackPlugin({
-			template: './src/index.html'
-		}),
-		new MiniCssExtractPlugin({
-			filename: '[name].bundle.css'
-		})
-	]
+  mode: "production",
+  entry: "./src/index.js",
+  output: {
+    filename: "[name].bundle.js",
```

Comment:
Add hash for fille names to invalidate cash in a browser when new version of the app is released https://webpack.js.org/guides/caching#output-filenames